docs-source/content/userguide/managing-domains/model-in-image/model-files.md

```diff
@@ -86,11 +86,11 @@ Some notes about the sample model file:
 
 Refer to this section if you need to control the order in which your model files are loaded.  The order is important when two or more model files refer to the same configuration, because the last model that's loaded has the highest precedence.
```

Comment:
same configuration - probably need to elaborate more >  e.g.  Same jdbc datasource URL is referenced by two different model files or the same property is referenced by two different properties files